providers/desec/desecProvider.go

```diff
@@ -0,0 +1,223 @@
+package desec
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
+	"github.com/StackExchange/dnscontrol/v3/pkg/printer"
+	"github.com/StackExchange/dnscontrol/v3/providers"
+	"github.com/miekg/dns/dnsutil"
+)
+
+/*
+desec API DNS provider:
+Info required in `creds.json`:
+   - auth-token
+*/
+
+// NewDeSec creates the provider.
+func NewDeSec(m map[string]string, metadata json.RawMessage) (providers.DNSServiceProvider, error) {
+	c := &api{}
+	c.creds.token = m["auth-token"]
+	if c.creds.token == "" {
+		return nil, fmt.Errorf("missing deSEC auth-token")
+	}
+
+	// Get a domain to validate authentication
+	if err := c.fetchDomainList(); err != nil {
+		return nil, err
+	}
+
+	return c, nil
+}
+
+var features = providers.DocumentationNotes{
+	providers.DocDualHost:            providers.Unimplemented(),
+	providers.DocOfficiallySupported: providers.Cannot(),
+	providers.DocCreateDomains:       providers.Can(),
+	providers.CanUseAlias:            providers.Cannot(),
+	providers.CanUseSRV:              providers.Can(),
+	providers.CanUseSSHFP:            providers.Cannot(),
```

Comment:
deSEC supports SSHFP

Reply:
Okay, this was something i was not sure about. I will update this.